example.con

```diff
@@ -0,0 +1,8 @@
+let A be a point;
+let B be a point;
+let C1 be a circle with center A and edge B;
```

Comment:
I think a clearer way to say this would be `circle with center A and radius B`

Otherwise there are multiple ways to say the same circle right? (not necessarily bad, but kind of weird)
`a circle with center A and edge -B` as well as the infinity many other points you could say